### PR TITLE
Send real frame times into Compose

### LINKED ISSUE #429: Send real frame time
It's needed for animations and such

### PR DESCRIPTION
**Pull request by JakeWharton**
Closes #429 

---

- [x] `CHANGELOG.md`'s "Unreleased" section has been updated, if applicable.

### CODE REVIEW

File: mosaic-runtime/src/commonMain/kotlin/com/jakewharton/mosaic/mosaic.kt [hunk 1]
```diff
@@ -70,12 +70,16 @@ public fun runMosaicBlocking(content: @Composable () -> Unit) {
 }
 
 public suspend fun runMosaic(content: @Composable () -> Unit) {
+	runMosaic(enterRawMode = true, content)
+}
+
+internal suspend fun runMosaic(enterRawMode: Boolean, content: @Composable () -> Unit) {
```
Comment (JakeWharton) [inline mosaic-runtime/src/commonMain/kotlin/com/jakewharton/mosaic/mosaic.kt:76]: I need `awaitComplete` on the tester, but what I really need is to go to bed so punting on that until tomorrow...

File: mosaic-runtime/src/linuxMain/kotlin/com/jakewharton/mosaic/platform.kt [hunk 2]
```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2023 Square, Inc.
```
Comment (JakeWharton) [inline mosaic-runtime/src/linuxMain/kotlin/com/jakewharton/mosaic/platform.kt:2]: All copied from Molecule

File: mosaic-runtime/src/commonMain/kotlin/com/jakewharton/mosaic/platform.kt [hunk 3]
```diff
@@ -9,3 +9,5 @@ internal expect inline fun AtomicBoolean.set(value: Boolean)
 internal expect inline fun AtomicBoolean.compareAndSet(expect: Boolean, update: Boolean): Boolean
 
 internal expect inline fun atomicBooleanOf(initialValue: Boolean): AtomicBoolean
+
+internal expect inline fun nanoTime(): Long
```
Comment (EpicDima) [inline mosaic-runtime/src/commonMain/kotlin/com/jakewharton/mosaic/platform.kt:13]: Alternatively, it was possible to use [TimeMark](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.time/-time-mark/) from Kotlin, which under the hood will use the most accurate time available in the system. There would be a little more code in commonMain, but without the code in the platform source sets.

Comment (JakeWharton) [reply mosaic-runtime/src/commonMain/kotlin/com/jakewharton/mosaic/platform.kt:13]: By capturing an instance and then calling `elapsedNow().inWholeNanoseconds`? My problem with that is it has to make this low-level call to get the current time and then do the substraction, pack the result into a double, and then we unpack the double. I'd rather just make the call directly.

Now once kotlinx.datetime's `Instant` moves to the stdlib... maybe we can switch to that and eliminate all this platform stuff.

**Pull request merged**